pkg/pluginmanager/manager_options.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package pluginmanager
+
+import (
+	"os"
+	"strings"
+
+	"github.com/vmware-tanzu/tanzu-cli/pkg/constants"
+
+	"github.com/vmware-tanzu/tanzu-plugin-runtime/log"
+)
+
+// PluginManagerOpts options to customize plugin lifecycle operations
+type PluginManagerOpts struct {
+	showLogs bool // Enable or disable logs
+}
+
+// GetLogMode sets the log mode based on the environment variable.
+func (p *PluginManagerOpts) GetLogMode() bool {
+	return p.showLogs
+}
+
+// SetLogMode sets the log mode based on the environment variable.
+func (p *PluginManagerOpts) SetLogMode() {
+	// Check if env is set which takes precedence
+	envLogMode := strings.ToLower(os.Getenv(constants.TanzuCLIShowPluginInstallationLogs))
+
+	switch envLogMode {
+	case "true":
+		log.QuietMode(false)
+		p.showLogs = true
+	case "false":
+		log.QuietMode(true)
+		p.showLogs = false
+	default:
+		log.QuietMode(!p.showLogs)
+	}
```

Comment:
Can we use something like below? This is the way we have been parsing the boolean environment variables in other places so it would be nice to keep it consistent. 
(ParseBool documentation: `It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False. Any other value returns an error`)

```
isTrue, _ := strconv.ParseBool(os.Getenv(constants.TanzuCLIShowPluginInstallationLogs)); 
if isTrue {
    // Enable Logs
} else{
    // Disable Logs
}
```

Reply:
Refactored the method to use strconv.ParseBool